x/staking/types/params.go

```diff
@@ -10,14 +10,22 @@ import (
 )
 
 const (
-	// defaultUnbondingTime reflects three weeks in seconds as the default
+	// DefaultUnbondingTime reflects three weeks in seconds as the default
 	// unbonding time.
-	defaultUnbondingTime time.Duration = 60 * 60 * 24 * 3 * time.Second
+	// TODO: Justify our choice of default here.
+	DefaultUnbondingTime time.Duration = time.Second * 60 * 60 * 24 * 3
+
+	// Default maximum number of bonded validators
+	DefaultMaxValidators uint16 = 100
+
+	// Default maximum entries in a UBD/RED pair
+	DefaultMaxEntries uint16 = 7
```

Comment:
I'm not sure I like the idea of having these values as constants as it encourages developers to reference these constants directly instead of referencing the `Params` object which calls the constants. If anything it could be reasonable to have a `DefaultParams` constant.